models/response_entities.go

```diff
@@ -19,14 +19,16 @@ type CreatedInsolvencyResourceLinks struct {
 
 // CreatedPractitionerResource is the entity returned in a successful creation of an practitioner resource
 type CreatedPractitionerResource struct {
-	IPCode          string                           `json:"ip_code"`
-	FirstName       string                           `json:"first_name"`
-	LastName        string                           `json:"last_name"`
-	TelephoneNumber string                           `json:"telephone_number"`
-	Email           string                           `json:"email"`
-	Address         CreatedAddressResource           `json:"address"`
-	Role            string                           `json:"role"`
-	Links           CreatedPractitionerLinksResource `json:"links"`
+	PractitionerId  string                       `bson:"practitioner_id"`
+	IPCode          string                       `json:"ip_code"`
+	FirstName       string                       `json:"first_name"`
+	LastName        string                       `json:"last_name"`
+	TelephoneNumber string                       `json:"telephone_number"`
```

Comment:
The example in the ticket didn't have it (perhaps because it's not a required field - the request spec says 'At least one of email or telephone_number must be supplied') but I don't think that's a reason to take it out - it is still listed in the spec at https://developer-specs.company-information.service.gov.uk/manipulate-company-data-api-filing/resources/practitioner?v=latest